INSTALL

```diff
@@ -806,7 +806,7 @@
                 possible to create your own ".conf" and ".tmpl" files and store
                 them locally, outside the OpenSSL source tree. This environment
                 variable can be set to the directory where these files are held
-                and will have Configure to consider them in addition to the
+                and will have Configure to consider them in preference to the
```

Comment:
I disagree, but perhaps it's my english.  I understand "in preference" that `Configure` would look at the indicated local config dir *instead of* the standard ones.  That is not the case.